tests_integration/__tests__/__snapshots__/write.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`do not write file with --write + formated file (stderr) 1`] = `""`;
 
 exports[`do not write file with --write + formated file (stdout) 1`] = `
-"formated.js[2K[1G[90m%s %dms[39m
+"formated.js%s %dms
```

Comment:
Should we open a new issue about the `%s` and `%d` here? Looks like a bug to me.

Reply:
#2960